app/src/components/ShareLinkButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Tooltip, TooltipTrigger } from "react-aria-components";
+import { useLocation } from "react-router";
+
+import { Button, Icon, Icons, Text, View } from "@phoenix/components";
+import { useNotifySuccess } from "@phoenix/contexts";
+
+export const ShareLinkButton = ({
+  successText,
+  tooltipText = "Copy link to clipboard",
+  preserveSearchParams = false,
+}: {
+  successText?: string;
+  tooltipText?: string;
+  preserveSearchParams?: boolean;
+}) => {
+  const location = useLocation();
+  const notifySuccess = useNotifySuccess();
+  return (
+    <TooltipTrigger delay={200}>
+      <Button
+        size="S"
+        variant="quiet"
+        leadingVisual={<Icon svg={<Icons.ExternalLinkOutline />} />}
```

Comment:
<img width="191" alt="Screenshot 2025-05-06 at 9 44 03 AM" src="https://github.com/user-attachments/assets/f94405c5-c746-4b70-b183-2cbb881fed36" />
I associate this visual to mean open in separate tab, not copy to clipboard.
I would either use the clipboard with text or use the share sheet icon
<img width="40" alt="Screenshot 2025-05-06 at 9 47 01 AM" src="https://github.com/user-attachments/assets/b957c33f-c252-446a-85c2-e395c5a2e295" />